docs/static/settings-file.asciidoc

```diff
@@ -178,6 +178,17 @@ Values other than `disabled` are currently considered BETA, and may produce unin
 | When set to `true`, quoted strings will process the following escape sequences: `\n` becomes a literal newline (ASCII 10). `\r` becomes a literal carriage return (ASCII 13). `\t` becomes a literal tab (ASCII 9). `\\` becomes a literal backslash `\`. `\"` becomes a literal double quotation mark. `\'` becomes a literal quotation mark.
 | `false`
 
+| `config.field_reference.escape_style`
+a| _EXPERIMENTAL_ setting that provides a way to reference fields that contain <<formal-grammar-escape-sequences,field reference special characters>> `[` and `]`.
```

Comment:
Should be "Technical preview" instead of "Experimental."  Reference: https://github.com/elastic/docs#using-the-technical-preview-admonition

I tried adding/formatting this nugget in a variety of ways.  So far, I don't really like any of them.  Here are two of several things I tried:

<img width="899" alt="Screen Shot 2022-05-20 at 4 31 18 PM" src="https://user-images.githubusercontent.com/35154725/169606266-85dfc953-44a6-4a9b-9c2f-e4d76e1ac803.png">

<img width="817" alt="Screen Shot 2022-05-20 at 4 39 51 PM" src="https://user-images.githubusercontent.com/35154725/169607501-96e851e5-68a7-4c3e-980e-147861a970e7.png">


Admonitions are supposed to handle formatting, but I haven't hit on a combination that looks good and provides adequate info for the user. Only tagging the option with "Preview" doesn't convey the risk that the option might change or go away. I can sync with @gtback next week for ideas and design intent. .